sia-gpu-miner.c

```diff
@@ -30,9 +38,8 @@ cl_mem nonceOutmobj = NULL;
 cl_kernel kernel = NULL;
 cl_int ret;
 
-CURL *curl;
-
-unsigned int blocks_mined = 0;
+size_t local_item_size = 256;
+unsigned int blocks_mined = 0, intensity = DEFAULT_INTENSITY;
```

Comment:
I think that, since these are full declarations, it makes sense to have this be two lines.